usage/app/model/MediaUsage.scala

```diff
@@ -1,131 +0,0 @@
-package model
```

Comment:
can we `git mv` this file to keep the history?

Reply:
done it, but not sure that PR is showing that correctly
I divide it to 3 commits to reflect that